{"uuid": "9e799637-d352-436b-a646-aa99d505b43c", "children": ["da7ad9eb-f8a7-439e-ac72-41ece8fbe99d"], "befores": [{"name": "_xunit_setup_class_fixture_Test_plugins", "status": "broken", "statusDetails": {"message": "socket.timeout: timed out\n", "trace": "  File \"D:\\python 3.8.7\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\python 3.8.7\\lib\\site-packages\\_pytest\\fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\python 3.8.7\\lib\\site-packages\\_pytest\\fixtures.py\", line 902, in call_fixture_func\n    fixture_result = next(generator)\n  File \"D:\\python 3.8.7\\lib\\site-packages\\_pytest\\python.py\", line 864, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"D:\\python 3.8.7\\lib\\site-packages\\_pytest\\python.py\", line 785, in _call_with_optional_argument\n    func(arg)\n  File \"d:\\wanwan1\\test_plugins.py\", line 19, in setup_class\n    self.p1.initialize()\n  File \"d:\\wanwan1\\login.py\", line 132, in initialize\n    self.tel_debug_FM104()\n  File \"d:\\wanwan1\\login.py\", line 58, in tel_debug_FM104\n    self.tn = telnetlib.Telnet(self.ip,self.port,timeout=10)\n  File \"D:\\python 3.8.7\\lib\\telnetlib.py\", line 218, in __init__\n    self.open(host, port, timeout)\n  File \"D:\\python 3.8.7\\lib\\telnetlib.py\", line 235, in open\n    self.sock = socket.create_connection((host, port), timeout)\n  File \"D:\\python 3.8.7\\lib\\socket.py\", line 808, in create_connection\n    raise err\n  File \"D:\\python 3.8.7\\lib\\socket.py\", line 796, in create_connection\n    sock.connect(sa)\n"}, "start": 1700720141953, "stop": 1700720151954}], "start": 1700720141953, "stop": 1700720152165}